corehq/apps/app_manager/models.py

```diff
@@ -1928,6 +1931,24 @@ def get_case_errors(self, needs_case_type, needs_case_detail, needs_referral_det
                 'module': module_info,
             }
 
+    def is_usercaseonly(self):
+        """
+        Return False is the usercase is unused, or if any forms update a
+        different case type. If the only case type updated in the module is
+        the usercase, return True.
+        """
+        def actions_use_another_case(actions):
+            return (actions.get('update_case', {'update': None})['update'] or
+                    actions.get('case_preload', {'preload': None})['preload'])
```

Comment:
These should check that the actions condition property is `never` since even if the action doesn't load or save any properties the case may still be loaded into the session: 

`action.condition.type == 'never'`